libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/ExecSpecRule/Conway.hs

```diff
@@ -391,38 +394,29 @@ instance
   signalSpec _ env st =
     txCertSpec env st
       <> constrained disableRegCerts
-      <> constrained disableDRepRegCerts
+      <> constrained disableDRepRegCerts'
     where
       disableRegCerts :: Term fn (ConwayTxCert Conway) -> Pred fn
       disableRegCerts cert =
-        (caseOn cert)
+        caseOn
+          cert
           ( branch $ \delegCert ->
-              (caseOn delegCert)
+              caseOn
+                delegCert
                 (branch $ \_ _ -> False)
                 (branch $ \_ _ -> True)
                 (branch $ \_ _ -> True)
                 (branch $ \_ _ _ -> True)
           )
-          (branch $ \_ -> True)
-          (branch $ \_ -> True)
-      -- ConwayRegDRep certificates seem to trigger some kind of a bug in the
-      -- MAlonzo code where it somehow reaches an uncovered case.
-      --
-      -- TODO investigate what's causing this bug and try to get rid of this
-      -- constraint
-      disableDRepRegCerts :: Term fn (ConwayTxCert Conway) -> Pred fn
-      disableDRepRegCerts cert =
-        (caseOn cert)
-          (branch $ \_ -> True)
-          (branch $ \_ -> True)
-          ( branch $ \govCert ->
-              (caseOn govCert)
-                (branch $ \_ _ _ -> False)
-                (branch $ \_ _ -> True)
-                (branch $ \_ _ -> False)
-                (branch $ \_ _ -> True)
-                (branch $ \_ _ -> True)
-          )
+          (branch $ const True)
+          (branch $ const True)
```

Comment:
```suggestion
          (branch $ \_ -> True)
```
I think it's also better to keep it like this for more consistency (I know hlint goes crazy about this stuff :s )